ui/package.json

```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "bootstrap": "^4.1.3",
     "react": "^16.5.1",
     "react-dom": "^16.5.1",
     "react-scripts": "1.1.5",
+    "reactstrap": "^6.5.0",
```

Comment:
The library with a bunch of cool components for us to use. https://reactstrap.github.io/